test/importing.coffee

```diff
@@ -2,20 +2,6 @@
 # ---------
 
 unless window? or testingBrowser?
-  test "coffeescript modules can be imported and executed", ->
```

Comment:
Okay, well, now please put it back 😄 Why exactly is this an issue? I don’t see a nested `test` call. And the following call is under the `unless window?` so clearly that isn’t an issue.